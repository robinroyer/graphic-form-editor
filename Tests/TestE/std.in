LIST
R r01 10 10 5 5
R r02 20 20 10 10
LIST
EXIT