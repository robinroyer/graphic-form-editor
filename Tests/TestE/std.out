OK
#New object: r01
OK
#New object: r02
R r01 10 10 5 5
R r02 20 20 10 10
